raft-tech/TANF-app
# 406 & 407: Fix focus on route change and alert on error

### This pull request changes...

Now when a user is routed to a new page, a screen reader will automatically read out the new `h1`. Also, if there is an alert that comes up (e.g. Network Error, Please Sign In) then a screen reader will now announce that alert.

**NOTE**
I am only opening this into raft-review right now. I want to work with it in the cloud environment and make sure there aren't any adverse effects from what I changed.

## FUNCTIONALITY
*Describe the functionality*
Addresses issues: #406 and #407 


## TO TEST
406
1. Go to `/edit-profile` and fill out the form.
2. Fill out form
3. Turn on screen reader
4. Click on `Request Access` and it should then announce the `h1` of the new page which is `Request Submitted`
407
1. Go to `/edit-profile` and fill out the form.
2. Open DevTools and set `Network` to `Offline`
3. Click on `Request Access`
4. It should read out `Network Error` while maintaining focus on the `Request Access` button. (I discussed whether this should be focused or simply read with Robert and we settled on just being read for now but happy to reassess).

### This pull request is ready to merge when...

  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)
